index.html

```diff
@@ -173,7 +276,7 @@ <h2>Contact me</h2>
                 <input type="email" name="email" placeholder="Email" required>
 
                 <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
-                <button type="submit">Get It Touch </button>
+                <button class="form-submit" type="submit">Get In Touch </button>
                 </form>
```

Comment:
- Great job in all 👍🏽, please make sure your contact form design matches the original Figma design
Your Design | Figma Design
![screenshot (1)](https://user-images.githubusercontent.com/44624138/171676909-5e4281ca-3d6d-4b70-8969-d01977d8bae7.png) | 
![screenshot](https://user-images.githubusercontent.com/44624138/171677016-88665580-5128-4d32-bd43-d66e30c5170a.png)